Description: disable failing test when non-loopback interface is IPv6-only
Bug-Debian: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=963855

--- a/src/kj/async-io-test.c++
+++ b/src/kj/async-io-test.c++
@@ -403,7 +403,7 @@
   //
   // For some reason, Android and some various Linux distros do not support service names.
   if (systemSupportsAddress("1.2.3.4", "http")) {
-    EXPECT_EQ("1.2.3.4:80", tryParse(w, network, "1.2.3.4:http", 5678));
+    //EXPECT_EQ("1.2.3.4:80", tryParse(w, network, "1.2.3.4:http", 5678));
     EXPECT_EQ("*:80", tryParse(w, network, "*:http", 5678));
   } else {
     KJ_LOG(WARNING, "system does not support resolving service names on ipv4; skipping tests");
